piapage/__init__.py

```
@@ -948,17 +949,17 @@ def build_catalog(incremental=True, verbose=True, download=False, path=None,
     if incremental:
         try:
-            with open(path) as f:
+            with open(path, 'rb') as f:
                 piapages = pickle.load(f)
 
             if verbose:
-                print 'catalog loaded'
+                print('catalog loaded')
 
             updating = True
         except:
```

Ruff (0.15.2)

[error] Do not use bare `except`

(E722)